bazel/external/quiche.BUILD

```diff
@@ -108,7 +108,10 @@ envoy_cc_library(
     name = "http2_constants_lib",
     srcs = ["quiche/http2/http2_constants.cc"],
     hdrs = ["quiche/http2/http2_constants.h"],
-    copts = quiche_copt,
+    copts = select({
+        "@envoy//bazel:windows_x86_64": [],
+        "//conditions:default": quiche_copt,
```

Comment:
W/o quiche_copt, can these targets be built in Windows? Do we still treat those warnings as error in Windows?

Reply:
Negative. They are invalid command line options, compilation will not proceed.

Reply:
Sorry, I meant, with this patch avoiding invalid command line options, we are able to compile, we aren't alerting on such warnings, although we aren't at the windows equiv of Wall Werror.

Reply:
Thanks for explanation!